app/__init__.py

```diff
@@ -1,36 +1,23 @@
 from flask import Flask
-from flask_sqlalchemy import SQLAlchemy
-from flask_migrate import Migrate
+from .db import db, migrate
+from .models import customer, rental, video
+from .routes.customer_routes import customers_bp
+from .routes.rental_routes import rentals_bp
+from .routes.video_routes import videos_bp
```

Comment:
To align with the conventions we show in class, I think we'd want to do:

```suggestion
from .routes.customer_routes bp as import customers_bp
from .routes.rental_routes import bp as rentals_bp
from .routes.video_routes import bp as videos_bp
```

Reply:
Since the blueprints aren't instantiated in the route files as-is, how do you feel about removing these imports in the scaffold? (I'd still leave behind the comment `# Register Blueprints here` below as a reminder to folks.)

Reply:
That works for me!